Protacon.RxMq.AzureServiceBus/Topic/AzureTopicPublisher.cs

```diff
@@ -86,14 +86,105 @@ public void Dispose()
                 .ToList()
                 .ForEach(x => x.Dispose());
         }
+
         public Task SendAsync<T>(T message) where T : new()
         {
             var topic = _settings.TopicNameBuilder(message.GetType());
 
             if (!_bindings.ContainsKey(topic))
-                _bindings.Add(topic, new Binding(_settings, _topicManagement, topic, typeof(T), _logger));
-
+            {
+                return TryCreateBinding(topic, typeof(T), message, 5, 60);
+            }
+            
             return _bindings[topic].SendAsync(message);
         }
+
+        /// <summary>
+        /// Creates new Binding for topic in a safe way. If initial tries fail, it will fallback to intervaled retry's.
+        /// </summary>
+        /// <param name="topic"></param>
+        /// <param name="type"></param>
+        /// <param name="message"></param>
+        /// <param name="instantRecoveryTries"></param>
+        /// <param name="lifeCycleRecoveryInterval"></param>
+        private async Task TryCreateBinding(string topic, Type type, object message, int instantRecoveryTries, int lifeCycleRecoveryInterval)
+        {
+            CancellationTokenSource cancellation = new CancellationTokenSource();
+            Binding binding;
+            int lifeCycleTryCount = 0;
+            Task task;
+
+            for (var i = 1; i <= instantRecoveryTries; i++)
+            {
+                _logger.LogInformation(
+                    $"TryCreateBinding: Try {i} of {instantRecoveryTries} for binding ('{topic}')");
+
+                binding = TryBinding(topic, type, message);
+                if (binding != null)
+                {
+                    _logger.LogInformation(
+                        $"TryCreateBinding: Binding successful ('{topic}', binding {i} of {instantRecoveryTries})");
+                    _bindings.Add(topic, binding);
+                    task = _bindings[topic].SendAsync(message);
```

Comment:
Can't this task variable to be removed? I think all usages of it can be replaced by simply await call directly.